docs/Data-Loading-For-Developers.md

```diff
@@ -0,0 +1,46 @@
+# Importing single data files for development
+In some cases, for example during development, it may be useful to import a single data file into an existing study. To import one data file at a time, you can use the following command. Note that this process will not validate the data.
```

Comment:
Might be useful to rephrase the word ‘development’ in this intro? Someone who isn't in our jargon circle would take it to mean something way, way broader than you intend.